### PR TITLE
Update imageEditor.js

### PR DESCRIPTION
**Pull request by OculusMode**
<ISSUE TYPE>Added method to set selection style
 feat: Added method to change selection styles.

### Description
Added method for user to access setSelectionStyles.

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

**Comment by OculusMode** [comment]
Im pretty new to git. please don't commit 2nd update about gitignore

File: src/js/imageEditor.js [hunk 1]
```diff
@@ -486,6 +486,16 @@ class ImageEditor {
...
+    /**
+    * set selection style properties
+    * @param {Object} styles - Object of styles
+    * @example
+    * imageEditor.setSelectionStyle({ cornerSize: 50 });
+    */
+    setSelectionStyle(styles) {
+        this._graphics.setSelectionStyle(styles);
+    }
+
```
Comment (junghwan-park) [inline src/js/imageEditor.js]: There is an option for setting `selectionStyle`.

I don't think this method is required.

File: src/js/imageEditor.js [hunk 2]
```diff
@@ -297,12 +297,12 @@ class ImageEditor {
...
         if ((e.ctrlKey || e.metaKey) && e.keyCode === keyCodes.Z) {
             // There is no error message on shortcut when it's empty
-            this.undo()['catch'](() => {});
+            this.undo()['catch'](() => { });
         }
 
         if ((e.ctrlKey || e.metaKey) && e.keyCode === keyCodes.Y) {
             // There is no error message on shortcut when it's empty
-            this.redo()['catch'](() => {});
+            this.redo()['catch'](() => { });
```
Comment (junghwan-park) [inline src/js/imageEditor.js]: There are two invalid spaces. Not suitable space for our ESLint rule.

File: .gitignore [hunk 3]
```diff
@@ -54,5 +54,3 @@ report
 *.vim
 test.html
 
-# Compiled files
-dist
```
Comment (junghwan-park) [inline .gitignore]: We managing bundle files by `production` branch.
I think you must not remove this line.

**Comment by junghwan-park** [comment]
@OculusMode 
If you have change your code and committed, please reopen this PR.

File: src/js/imageEditor.js (hunk 1 above)
Comment (OculusMode) [reply src/js/imageEditor.js]: @junghwan-park indeed there is but only if you include ui. If you don't include UI. then there is no way to change selectionStyles i.e. cornerSize.
Hence one can use this method to do something like:


```
imageEditor.setSelectionStyle({ cornerSize: 10 });
```

**Comment by OculusMode** [comment]
@junghwan-park I have made changes in my current master, can you please re open?

**Pull request closed without merging**